src/com/company/Main.java

```diff
@@ -1,46 +1,82 @@
+    private static File fileToRead;
+    private static File fileToSave;
+    public static Scanner scanner;
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) {
 
-        System.out.println("Path to unsorted numbers file:\n");
+        scanner = new Scanner(System.in);
+        String fileNameToRead = getFirstParam("Specify path to file with unsorted numbers", scanner);
+        String directoryNameToSaveSortedData = getDirectoryToSave("Specify path to directory to save sorted data", scanner);
+        String fileNameToSaveSortedData = getFileNameToSave("Specify file name to save sorted data", scanner);
+        validateFiles(fileNameToRead, directoryNameToSaveSortedData ,fileNameToSaveSortedData);
+        FileUtils.getUnsortedNumbers(fileToRead);
+        Sorting.selectSortingMethod();
+        FileUtils.saveSortedNumbers(Sorting.numbersToSave, fileToSave);
+    }
 
-        try{
-            Scanner filePathScanner = new Scanner(System.in);
-            String filePathAndName = filePathScanner.nextLine();
-            System.out.println("you entered path: " + filePathAndName + "\n");
-            fileToRead = new File(filePathAndName);
+    private static String getFirstParam (String messageUserCanSeeInConsole, Scanner scanner) {
+        System.out.println(messageUserCanSeeInConsole);
+        return scanner.nextLine();
+    }
+    private static String getDirectoryToSave(String messageUserCanSeeInConsole, Scanner scanner){
+        System.out.println(messageUserCanSeeInConsole);
+        return scanner.nextLine();
+    }
+    private static String getFileNameToSave(String messageUserCanSeeInConsole, Scanner scanner){
+        System.out.println(messageUserCanSeeInConsole);
+        return new StringBuilder().append(scanner.nextLine()).append(".txt").toString();
+    }
+
+    private static void validateFiles(String fileNameToRead, String directoryNameToSaveSortedData, String fileNameToSaveSortedData){
+        try {
+            fileToRead = new File(fileNameToRead);
+            fileToSave = new File(directoryNameToSaveSortedData, fileNameToSaveSortedData);
 
-            System.out.println("Enter file name for saving sorted numbers ");
-            Scanner fileSaveScanner = new Scanner(System.in);
-            String fileSaveSorted = fileSaveScanner.nextLine();
-            fileToSave = new File(fileSaveSorted + ".txt");
+            boolean result = fileToSave.createNewFile();
+            if(result){
+                System.out.println("File to save sorted numbers is created: "+fileToSave.getCanonicalPath());
+            }
+            else {
+                System.out.println("File already exist at location: "+fileToSave.getCanonicalPath());
+            }
 
+            if (!fileToRead.exists()) {
+                System.out.println("\'" + fileNameToRead + "\' file is not found. Check path and file name");
+                System.exit(0);
```

Comment:
Do not use exit. Read exception throwing.

Reply:
deleted exit()